### DISCUSSION TITLE
调试源码的时候发现两个问题，自行解决几个小时没有答案，希望得到解答，谢谢

### DISCUSSION THREAD

**Discussion by cnhekai** [element-plus/element-plus#18757] [Q&A]
### 运行`pnpm dev` 的时候css变量报错（这个问题我解决了，想知道这样解决是不是不太合适）
如图：
![image](https://github.com/user-attachments/assets/48240f71-550b-4d16-bb12-155d08183d80)
解决方案，在`packages\theme-chalk\src\mixins\mixins.scss`下添加如下代码:
![image](https://github.com/user-attachments/assets/3e260b27-2ce0-43da-8645-0774fc408537)

### 运行`pnpm t` 的时候，没有改变任何代码，但是有几个测试用例没通过，请问为什么有这样
如图：
![image](https://github.com/user-attachments/assets/a78e47c2-3e47-4fc2-bdf0-958008bd188e)

**Comment by warmthsea** [comment]
1、就是如此解决
2、确保你使用 pnpm@9.5.0 安装依赖，以及 node 版本 >18

**Reply by cnhekai** [reply]
感谢您的回复，我已经成功的解决了相关问题，另外我在`CONTRIBUTING.md`文件中添加了pnpm的版本描述，用于方便后来者更快的解决问题。已提交[pr](https://github.com/element-plus/element-plus/pull/18769)
